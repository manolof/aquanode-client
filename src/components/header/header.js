import React, { PureComponent } from 'react';

class Header extends PureComponent {
	render() {
		return (
			<div className="container">
				<h1>
					Aquanode
					<span role="img" aria-label="Fish">🐟</span>
					<span role="img" aria-label="Plant">🌿</span>
				</h1>
			</div>
		);
	}
}

export {
	Header,
};
