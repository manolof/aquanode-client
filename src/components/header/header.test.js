import { shallow } from 'enzyme';
import React from 'react';

import { Header } from './header';

describe('Header Component', () => {
	it('should render', () => {
		const wrapper = shallow(<Header/>);

		expect(wrapper.getElements()).toMatchSnapshot();
	});
});
