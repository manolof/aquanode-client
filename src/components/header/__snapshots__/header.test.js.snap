// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component should render 1`] = `
Array [
  <div
    className="container"
  >
    <h1>
      Aquanode
      <span
        aria-label="Fish"
        role="img"
      >
        🐟
      </span>
      <span
        aria-label="Plant"
        role="img"
      >
        🌿
      </span>
    </h1>
  </div>,
]
`;
